stacker/providers/aws/default.py

```diff
@@ -362,30 +363,63 @@ def create_change_set(cfn_client, fqn, template, parameters, tags,
     return changes, change_set_id
 
 
+def check_tags_present(actual, expected):
+    actual_set = set((item["Key"], item["Value"]) for item in actual)
+    expected_set = set((item["Key"], item["Value"]) for item in expected)
+
+    return actual_set >= expected_set
```

Comment:
It's ok if the actual set has items the expected set doesn't?

Reply:
Won't it always have at least the automatically generated CloudFormation tags?